OracleTuxedo/core/samples/shm/simpapp_runme.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+#
+# Shell script to build and run simpapp.  It assume TUXDIR has been set, or that 
+# Tuxedo has been installed to: ~/tuxHome/tuxedo12.1.3.0.0   If not, invoke 
+# this script with a single argument indicating the location of TUXDIR.
+#
+# Author: Todd Little
+#
+# Usage: source simpapp_runme.sh [TuxedoVersion]
+#        TuxedoVersion: 12.1.3 or 12.2.2, 12.2.2 by default
+#
+cd /u01/oracle/user_projects
+rm -rf simpapp
+mkdir simpapp
+cd simpapp
+if [ ! -z "$1" ]
+    then
+  if [ "$1" = "12.1.3" ]
+  then
+	export TUXDIR=~/tuxHome/tuxedo12.1.3.0.0
```

Comment:
Isn't `$TUXDIR` set in the parent image? You shouldn't need to work out what it is downstream.